src/test/java/ejercicio/banco/repository/MySqlAccountRepositoryTest.java

```diff
@@ -0,0 +1,96 @@
+package ejercicio.banco.repository;
+
+import ejercicio.banco.dto.Account;
+import ejercicio.banco.util.DataBaseUtil;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class MySqlAccountRepositoryTest {
+    private static final String TABLE = "bank_project_test";
+    private static final String NOTHING = "nothing";
+    private static final String EMPTYTABLE = "empty";
+    private List<Account> accounts = new ArrayList();
+    private List<Account> empty = new ArrayList();
+    private Account account;
+    private Account expectedAccount1 = new Account(1, "Peter",(float)3.00, "PIPIRANA87");
+    private Account expectedAccount2 = new Account(2, "Aurelio", (float)8.49, "SATURN15STINKS");
+    private List<Account> full = new ArrayList();
+
+    @Mock
+    private DataBaseUtil util;
```

Comment:
We will discuss about the tests later on because they are not working now (they try to establish a connection with the real database and that is, conceptually, wrong). Remember Unit Test scope is to test the class in isolation so no real connection should be established.

Reply:
ok